src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7403,4 +7400,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureLambdaParamsArray" xml:space="preserve">
     <value>lambda params array</value>
   </data>
+  <data name="WRN_ParamsArrayInLambdaOnly" xml:space="preserve">
+    <value>Parameter {0} has params modifier in lambda but not in target delegate type.</value>
+  </data>
+  <data name="WRN_ParamsArrayInLambdaOnly_Title" xml:space="preserve">
+    <value>The params modifier is present in lambda but not in target delegate type.</value>
```

Comment:
>The params modifier is present in [](http://example.com/codeflow?start=11&length=33)

Consider using the same text as the warning, without the placeholder, for consistency: "Parameter has params modifier in lambda ..."

Reply:
Thanks, changed: fb140077eb16f395631cf1bfa6e7c9fdb277bfd4